logrotate.c

```diff
@@ -2464,18 +2464,19 @@ int main(int argc, const char **argv)
 
     struct poptOption options[] = {
 	{"debug", 'd', 0, 0, 'd',
-	 "Don't do anything, just test (implies -v)"},
-	{"force", 'f', 0, &force, 0, "Force file rotation"},
+	 "Don't do anything, just test (implies -v)", NULL},
+	{"force", 'f', 0, &force, 0, "Force file rotation", NULL},
```

Comment:
Not a problem, but a warning =Wall and -Wextra are in use: missing an initializer. Sample warning is added to commit message.